frontend/app/about/page.tsx

```diff
@@ -105,100 +154,90 @@ const About = () => {
-                      <h3 className='mt-6 text-2xl font-semibold'>{member.name}</h3>
-                      <div className='flow-root'>
-                        <p className='float-left text-2xl'>{member.gradYear}</p>{' '}
-                        <p className='float-right'>
-                          {navigation.map((item) => (
-                            <a
-                              key={item.name}
-                              href={member.link}
-                              className='text-gray-400 hover:text-gray-500'
-                              target='_blank'
-                              rel='noopener noreferrer'
-                            >
-                              <span className='sr-only'>{item.name}</span>
-                              <item.icon className='h-6 w-6' aria-hidden='true' />
-                            </a>
-                          ))}
-                        </p>
-                      </div>
-                    </div>
-                  ))} */}
-                {/* </div> */}
+              </div>
+
+              {/* Team Leads Section */}
+              <div className="mt-16">
+                <h2 className="text-3xl font-bold text-center text-[var(--system-text-color)] mb-4">
+                  Team Leads
+                </h2>
+                <p className="text-center text-lg mb-8 text-[var(--system-text-color)]">
+                  Development for HoagiePlan continued as part of Hoagie. Issac and Gabriel are <br />
+                  managing HoagiePlan's current Developers.
+                </p>
+                <div className="flex flex-wrap justify-center gap-6">
+                  {teamLeads.map((member, index) => (
+                    <TeamMemberCard 
+                      key={member.name} 
+                      member={member} 
+                      index={foundingTeam.length + index} 
+                    />
+                  ))}
+                </div>
+              </div>
+
+              {/* Contributors Section */}
+              <div className="mt-16">
+                <h2 className="text-3xl font-bold text-center text-[var(--system-text-color)] mb-4">
+                  Contributors
+                </h2>
+                <p className="text-center text-lg mb-8 text-[var(--system-text-color)]">
+                  HoagiePlan would not exist without our talented team of Developers.
+                </p>
+                <ul className="list-disc text-center text-lg text-[var(--system-text-color)] max-w-2xl mx-auto space-y-4">
+                  {contributors.map((contributor) => (
+                    <li key={contributor.name} className="list-inside">
+                      <a
+                        href={contributor.link}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        className="hover:text-blue-500 transition-colors"
+                      >
+                        {contributor.name}
```

Comment:
Let's make the bullet points all align on the left for a more consistent styling